cmd/extensions/installed/cmd.go

```diff
@@ -1,34 +1,34 @@
-package installed
-
-import (
-	"fmt"
-
-	"github.com/spf13/cobra"
-)
-
-// InstalledCmd represents the installed command
-var InstalledCmd = &cobra.Command{
-	Use:   "installed",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("installed called")
-	},
-}
-
-func init() {
-	// Here you will define your flags and configuration settings.
-
-	// Cobra supports Persistent Flags which will work for this command
-	// and all subcommands, e.g.:
-	// InstalledCmd.PersistentFlags().String("foo", "", "A help for foo")
-
-	// Cobra supports local flags which will only run when this command
-	// is called directly, e.g.:
-	// InstalledCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
-}
+package installed
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// InstalledCmd represents the installed command
+var InstalledCmd = &cobra.Command{
+	Use:   "installed",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("installed called")
+	},
+}
+
+func init() {
+	// Here you will define your flags and configuration settings.

```

Comment:
No comments please. Let's be vigilant to not push any half backed code. 
This is WIP but flagging it early.